src/Controller/HomeController.php

```diff
@@ -2,27 +2,58 @@
 
 namespace App\Controller;
 
-use App\Contributors\Service\Repository as ContributorsRepository;
-use App\Contributors\Model\Contributor;
```

Comment:
Why you remove logic for get contributors?

Reply:
Probably a bad conflict resolution, I'll check